modules/datafusion/main.tf

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  prefix_length      = 22
+  address_type       = "INTERNAL"
+  porpuse            = "VPC_PEERING"
```

Comment:
typo (`s/porpuse/purpose/`), but I don't see a need for a local here, just set the value directly in the resource attribute; same for `address_type`.

Reply:
removed.